### DISCUSSION TITLE
Why does vscode not show any output for very simple C++ code?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-cpptools#11274] [Q&A]
I can't seem to figure out why this code isn't producing any output. I don't even know where to begin to fix it. any suggestions?
![image](https://github.com/microsoft/vscode-cpptools/assets/122657606/e642d16f-5096-4180-b622-2c17d5c1dc6b)

**Comment by sean-mcmanus** [comment]
How are you building/running the main.exe process?
